### ISSUE TITLE
Bug Report: Alarm goes to unknown state when disarming via the panel

### ISSUE THREAD

**Issue by rdeutsch3** [bloomkd46/homebridge-XfinityHome#35]
### Describe The Bug:

When disarming the alarm via the panel, the plugin goes to unknown state in HK.  If the alarm is disabled in HK, everything is fine and it transitions to disarmed.

### To Reproduce:

Disarm via panel on wall.

### Expected behavior:

Alarm transitions to disarmed state.

### Logs:

```shell
[14/11/2023, 02:35:17] [Xfinity Home System] Removed 0 Accessories
[14/11/2023, 07:54:54] [Xfinity Home System] Panel: Disarming... 
[14/11/2023, 07:54:54] [Xfinity Home System] Panel: Disarmed 
[14/11/2023, 07:55:04] [Xfinity Home System] Entry Doors: Opened

See the corresponding HK Activity History in the comment below.
```

### Config:

```shell
{
    "bridge": {
        "name": "Homes-Homebridge",
        "username": "ZZZZZ",
        "port": 51035,
        "pin": "ZZZZ",
        "advertiser": "ciao",
        "bind": []
    },
    "accessories": [],
    "platforms": [
        {
            "name": "UI Config",
            "port": 8581,
            "auth": "form",
            "theme": "auto",
            "tempUnits": "f",
            "lang": "auto",
            "platform": "config"
        },
        {
            "name": "Xfinity Home System",
            "pin": "1597",
            "refreshToken": "ZZZZ",
            "temperatureSensors": true,
            "logLevel": 3,
            "hideUnsupportedDeviceWarnings": false,
            "logWatchdogErrors": false,
            "_bridge": {
                "username": "ZZZZZ",
                "port": 32064
            },
            "platform": "XfinityHomePlatform"
        }
    ]
}
```

### Operating System:

Raspberian

### Node Version:

v20.9.0

### NPM Version:

10.1.0

### Homebridge Version:

1.7.0

### Plugin Version:

v3.4.1

**Comment by rdeutsch3** [comment]
Here is the corresponding Activity log from the Security section in Apple Home:
![image](https://github.com/bloomkd46/homebridge-XfinityHome/assets/119368223/7fe93d54-26f3-4d40-ba83-187460696aa8)


The subtle subsequent issue is when the alarm goes to unknown state, the automation that should switch my cameras to disarmed also goes unknown (and I haven't figured out a way around this, yet).

**Comment by Jrpadg** [comment]
Hello rdeutch3

I am sorry that I cannot help you with your issue, but I wondered if you might have any advice in me trying to connect Homebridge to Xfinity Home.  I have followed all steps, but I can’t get the proxy tool to work.  If you have any suggestions it would be awesome!  Thank you

**Comment by rdeutsch3** [comment]
> Hello rdeutch3
> 
> I am sorry that I cannot help you with your issue, but I wondered if you might have any advice in me trying to connect Homebridge to Xfinity Home. I have followed all steps, but I can’t get the proxy tool to work. If you have any suggestions it would be awesome! Thank you

Unfortunately I don’t have any advice other than follow the instructions exactly on your phone.  I found it would not work at all on my Synology HB (as I recall due to assumptions in folder paths), but worked fine on my parallel Raspberry Pi installation.  Good luck.

**Comment by rdeutsch3** [comment]
Is this fixed with v 3.6.0?

**Comment by bloomkd46** [comment]
Potentially, but if not, please help me out by going into `Xfinity Home Settings`>`Logs`>`Panel` and let me know the status changes when it’s disarming.
![IMG_7031](https://github.com/user-attachments/assets/6777f902-0ca9-429e-a2ee-a87ebe7f5947)

**Comment by rdeutsch3** [comment]
Will do in the am.

**Comment by rdeutsch3** [comment]
I’ll say “much better”.  It no longer goes to “unknown” and my automations are much happier.  However there was a significant pause (15-30s?) between when the actual alarm armed and more concerning the Apple Home app displayed the alarm panel as unresponsive before it flipped over to armed.  There is no pause when disarming, the alarm panel on the wall and Apple Home are a few seconds apart.
Here is a screenshot of the log showing the disarm test from my wife, and me arming it a few minutes later.
![image](https://github.com/user-attachments/assets/1cebcc6f-22a9-4e20-b60f-cfef12721949)

**Comment by rdeutsch3** [comment]
Had a funky transition from Off to Arming tonight.  I’m still on 3.6.1-beta2
![image](https://github.com/user-attachments/assets/acc4aa28-3417-4a94-a10e-7a852184b7d1)
The opened/closed didn’t happen in the real world.  I think it was the result of an underlying/unreported error, but cannot say.
It did throw this error later as it was arming
<img width="885" alt="image" src="https://github.com/user-attachments/assets/3a10c750-ecd7-4607-9d15-21641468ee0c">

Then it armed.
![image](https://github.com/user-attachments/assets/b9556420-e092-4847-b89f-e91a2599a50f)

**Comment by rdeutsch3** [comment]
Feels like it makes sense to propagate 3.6.1.-beta.4 to release?  I've been running it for months and it seems stable.

**Comment by bloomkd46** [comment]
Yes, sorry for the delay, I forgot I left it in Beta. I  just released it as `v3.6.2`

**Comment by rdeutsch3** [comment]
Thank you.  Didn't know if you were still investigating something or not.
